### ISSUE TITLE
Missing tolino firmwares

### ISSUE THREAD

**Issue by Liriel-666** [pgaskin/KoboStuff#53]
The messing tolino firmware are herein to find
https://mytolino.de/software-updates-tolino-ereader/

**Comment by pgaskin** [comment]
Yep, I know how to check for tolino versions directly, but I haven't updated the kfwproxy backend to support this yet. I'll do it when I have time later this year.
